wagit/main.cpp

```diff
@@ -40,7 +47,8 @@ int main(int, char* argv[]) {
     }
     else if (command == "init")
```

Comment:
Store these strings in variables. And maybe we can use switch case or some other smart way of doing this instead of "if else"?

Reply:
Switch doesn't work for strings. We could if-else through an enum but that's an extra pointless level of abstraction.
And these strings aren't going to change, so keeping them here should be fine.

Reply:
I think we should have something like a class GitCommands where we can maintain all of these. It would make it easier to see the available list of commands too. And would help us with writing the error message and stuff too.

Reply:
Hmm, cool. Let's do that in a later PR.
This is anyway the front end, not a part of the core functionality